Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mhart/alpine-node:5.7.1
+
+# The working directory must be set using WORKDIR
+WORKDIR .
+ENV HOME="/"
+
+# If you have native dependencies, you'll need extra tools. Also remove any cached files from the installs
+# RUN apk add --no-cache make gcc g++ python
+
+ADD ./package.json ./
+
+# If don't you need npm, use a base tag
+RUN npm install --production
```

Comment:
You also want to add 
`ENV NODE_ENV production`
Some packages act differently (and some emit kind warnings on what not to do in prod; `express-session` comes to mind)